src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -306,6 +307,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ),
                     'required' => false,
                     'invalid_message' => $this->trans('This field is invalid.', 'Admin.Notifications.Error'),
+                    'constraints' => [
+                        new GreaterThan([
```

Comment:
```suggestion
                        new GreaterThanOrEqual([
```

Reply:
I agree with Pierre, with the current code you can't enter 0 anymore which is a valid value

Reply:
@PierreRambaud good catch !